### PR TITLE
Feat/richtext

### PR DESCRIPTION
**Pull request by alinurr1**
## Description of the change
This PR adds support of `<br/>` tag to RichText component

## Testing the change

- [ ] Write your testing instructions here

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Non-Breaking Change (change to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [x] The code changed/added as part of this pull request has been covered with tests
- [x] All tests related to the changed code pass in development

### Code review

- [x] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached.

### CODE REVIEW

File: .changeset/thirty-timers-knock.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@localyze-pluto/components": patch
```
Comment (jvaladas) [inline .changeset/thirty-timers-knock.md:2]: Is this a patch or a feature?

File: .changeset/thirty-timers-knock.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+add br tag support to RichText
```
Comment (jvaladas) [inline .changeset/thirty-timers-knock.md:5]: Should this start with a capital letter? 
```suggestion
Add br tag support to RichText
```

**Review by jvaladas — APPROVED** [review]
Looks good to me, left two questions

File: .changeset/thirty-timers-knock.md (hunk 1 above)
Comment (alinurr1) [reply .changeset/thirty-timers-knock.md:2]: IMO it's a patch, because it's very small change and so it's not `minor` or `major` bump

Comment (jvaladas) [reply .changeset/thirty-timers-knock.md:2]: Ok, thank you, I still need to get used to the naming

**Pull request merged**